modules/installation-configuration-parameters.adoc

```diff
@@ -766,6 +766,19 @@ ifdef::rhv[]
 |`platform.ovirt.ingress_vip`
 |Required. An IP address on the machine network that will be assigned to the Ingress virtual IP (VIP).
 |String. Example: `10.46.8.232`
+
+|`platform.ovirt.affinityGroups.description`
+|Required. A description of the affinity group.
+|String. Example: `AffinityGroup for spreading each compute machine to a different host`
+|`platform.ovirt.affinityGroups.enforcing`
+|Required. Defines if the rule is enforced. `true` - the rule is enforce. `false` the rule is not enforced.
+|String. Example: `true`
+|`platform.ovirt.affinityGroups.name`
+|Required. A name for the affinity group. By default, this is either `compute` for worker nodes or 'controlPlan' for control nodes.
```

Comment:
Remove `By default, this is either `compute` for worker nodes or 'controlPlan' for control nodes.`

Reply:
Fixed